### PR TITLE
#EE-21892|Counter Component Missing Error States

### PR DESCRIPTION
**Pull request by boriskaWix**
This PR contains changes to Counter component. 
changes include: a tooltip that is shown if the user chooses to show an error message while the component is in error state.
Related jira [here](https://jira.wixpress.com/browse/EE-21892)

### CODE REVIEW

**Comment by jonathanadler** [comment]
Hey @johannaasseraf, can you take a look and give a design review?
Thanks!

**Comment by johannaasseraf** [comment]
@boriskaWix where did you get the icon from?

**Comment by boriskaWix** [comment]
> @boriskaWix where did you get the icon from?

it's from wix-ui-icons-common

File: src/components/Counter/Counter.tsx [hunk 1]
```diff
@@ -5,6 +5,9 @@ import style from './Counter.st.css';
 import { ReactComponent as Plus } from '../../assets/icons/plus.svg';
 import { ReactComponent as Minus } from '../../assets/icons/minus.svg';
 import { TPAComponentProps } from '../../types';
+import { Tooltip } from '../Tooltip';
+import { TooltipSkin } from '../Tooltip/TooltipEnums';
+import StatusAlertSmall from 'wix-ui-icons-common/StatusAlertSmall';
```
Comment (jonathanadler) [inline src/components/Counter/Counter.tsx]: You should use our Error Icon instead:
```js
import { ReactComponent as Error } from '../../assets/icons/Error.svg';
```

Comment (jonathanadler) [reply src/components/Counter/Counter.tsx]: @boriskaWix ?

**Comment by jonathanadler** [comment]
@johannaasseraf 
Icon was updated, but i'm not sure it looks they way it should.
https://wix-wix-ui-tpa-pr-265.surge.sh/?activeTab=Playground&path=%2Fstory%2Fcomponents--counter
I think we also might need to give it a larger default minimum width.

What do you think?

**Comment by johannaasseraf** [comment]
> @johannaasseraf
> Icon was updated, but i'm not sure it looks they way it should.
> https://wix-wix-ui-tpa-pr-265.surge.sh/?activeTab=Playground&path=%2Fstory%2Fcomponents--counter
> I think we also might need to give it a larger default minimum width.
> 
> What do you think?

It looks good.
Lets change the min width to 140px
And the error message to "This is an error message"

**Comment by jonathanadler** [comment]
> > @johannaasseraf
> > Icon was updated, but i'm not sure it looks they way it should.
> > https://wix-wix-ui-tpa-pr-265.surge.sh/?activeTab=Playground&path=%2Fstory%2Fcomponents--counter
> > I think we also might need to give it a larger default minimum width.
> > What do you think?
> 
> It looks good.
> Lets change the min width to 140px
> And the error message to "This is an error message"

Cool.
@boriskaWix please fix when you get a chance.
Thanks!

**Comment by johannaasseraf** [comment]
> > > @johannaasseraf
> > > Icon was updated, but i'm not sure it looks they way it should.
> > > https://wix-wix-ui-tpa-pr-265.surge.sh/?activeTab=Playground&path=%2Fstory%2Fcomponents--counter
> > > I think we also might need to give it a larger default minimum width.
> > > What do you think?
> > 
> > 
> > It looks good.
> > Lets change the min width to 140px
> > And the error message to "This is an error message"
> 
> Cool.
> @boriskaWix please fix when you get a chance.
> Thanks!

It's just a min width for storybook right? each tpa will give it's own width?

**Comment by jonathanadler** [comment]
> > > > @johannaasseraf
> > > > Icon was updated, but i'm not sure it looks they way it should.
> > > > https://wix-wix-ui-tpa-pr-265.surge.sh/?activeTab=Playground&path=%2Fstory%2Fcomponents--counter
> > > > I think we also might need to give it a larger default minimum width.
> > > > What do you think?
> > > 
> > > 
> > > It looks good.
> > > Lets change the min width to 140px
> > > And the error message to "This is an error message"
> > 
> > 
> > Cool.
> > @boriskaWix please fix when you get a chance.
> > Thanks!
> 
> It's just a min width for storybook right? each TPA will give its own width?

No,
We need to give it a min-width in order for the input and the icon to live peacefully.
Or, we can change the width when there's an error icon, but then there will be a jump.

**Comment by johannaasseraf** [comment]
> > > > > @johannaasseraf
> > > > > Icon was updated, but i'm not sure it looks they way it should.
> > > > > https://wix-wix-ui-tpa-pr-265.surge.sh/?activeTab=Playground&path=%2Fstory%2Fcomponents--counter
> > > > > I think we also might need to give it a larger default minimum width.
> > > > > What do you think?
> > > > 
> > > > 
> > > > It looks good.
> > > > Lets change the min width to 140px
> > > > And the error message to "This is an error message"
> > > 
> > > 
> > > Cool.
> > > @boriskaWix please fix when you get a chance.
> > > Thanks!
> > 
> > 
> > It's just a min width for storybook right? each TPA will give its own width?
> 
> No,
> We need to give it a min-width in order for the input and the icon to live peacefully.
> Or, we can change the width when there's an error icon, but then there will be a jump.

ok so lets do 104px min width

**Comment by jonathanadler** [comment]
@johannaasseraf 
can you take a look to see that it's ok please 🙏 
https://wix-wix-ui-tpa-pr-265.surge.sh/?path=/story/components--counter

**Pull request merged**